src/main/java/gov/acwi/wqp/etl/result/TransformBiodataResult.java

```diff
@@ -0,0 +1,103 @@
+package gov.acwi.wqp.etl.result;
+
+import gov.acwi.wqp.etl.Application;
+import gov.acwi.wqp.etl.biodata.biodataResult.BiodataBiodataResult;
+import gov.acwi.wqp.etl.biodata.biodataResult.BiodataBiodataResultRowMapper;
+import org.springframework.batch.core.Step;
+import org.springframework.batch.core.configuration.annotation.StepBuilderFactory;
+import org.springframework.batch.core.job.builder.FlowBuilder;
+import org.springframework.batch.core.job.flow.Flow;
+import org.springframework.batch.core.job.flow.support.SimpleFlow;
+import org.springframework.batch.core.step.tasklet.Tasklet;
+import org.springframework.batch.item.ItemProcessor;
+import org.springframework.batch.item.ItemWriter;
+import org.springframework.batch.item.database.BeanPropertyItemSqlParameterSourceProvider;
+import org.springframework.batch.item.database.ItemSqlParameterSourceProvider;
+import org.springframework.batch.item.database.JdbcBatchItemWriter;
+import org.springframework.batch.item.database.JdbcCursorItemReader;
+import org.springframework.batch.item.database.builder.JdbcCursorItemReaderBuilder;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.io.Resource;
+import org.springframework.util.FileCopyUtils;
+
+import javax.sql.DataSource;
+import java.io.IOException;
+
+@Configuration
+public class TransformBiodataResult {
+
+    @Autowired
+    @Qualifier("biodataResultProcessor")
+    private ItemProcessor<BiodataBiodataResult, BiodataBiodataResult> processor;
+
+    @Autowired
+    private StepBuilderFactory stepBuilderFactory;
+
+    @Autowired
+    @Qualifier(Application.DATASOURCE_BIODATA_QUALIFIER)
+    private DataSource dataSourceBiodata;
+
+    // Truncate the biodata.result table before populating so we can restart the unique row number identity
```

Comment:
comment seems misleading - we are truncating and resetting, not truncating so we can restart.
several ways to have an empty table with a fresh set of identity values...